ansible/module_utils/multi_servers_utils.py

```diff
@@ -0,0 +1,63 @@
+class MultiServersUtils:
```

Comment:
looks like there are 2 MultiServersUtils class with almost the same content, is this expected?

Reply:
Yes, there are two class MultiServersUtils,
one in ansible/module_utils/multi_servers_utils.py
another in ansible/plugins/filter/filters.py

module_utils will be copied to **remote host,** while filter plugin will be loaded at **local host**,

they run in different environment, so they can't refer each other even the relative import is not work.

Reply:
this is interesting.... it might be better to update the file name to make it more explicit, since this will be really weird to read.